tools/pepsi/src/power_off.rs

```diff
@@ -14,15 +14,25 @@ pub struct Arguments {
 }
 
 pub async fn power_off(arguments: Arguments) -> anyhow::Result<()> {
-    let tasks = arguments.naos.into_iter().map(|nao_address| async move {
-        let nao = Nao::new(nao_address.ip);
-        nao.power_off()
-            .await
-            .with_context(|| format!("Failed to power {nao_address} off"))
+    let multi_progress = ProgressIndicator::new();
+
+    let tasks = arguments.naos.into_iter().map(|nao_address| {
+        let multi_progress = multi_progress.clone();
```

Comment:
This cloning looks weird. Couldn't you just remove the cloning and directly get a task that you afterwards hand into the async task? Then there is no cloning involved in this location. You could also remove the `#[derive(Clone)]` from the multi progress because you probably want to have singleton behavior for this struct, right?

Reply:
I need the cloned `multi_progress` because it is `move`d into the `async move` block. If I move the line with `let progress = multi_progress.task(...)` outside of the `async{}` block, we sometimes observe "zombie" progress bar lines probably caused by some race condition.

Reply:
The "zombie" progress bar lines are now fixed and the cloning removed.